build.gradle.kts

```diff
@@ -23,6 +23,11 @@ plugins {
 	id("com.github.johnrengelman.shadow") version "5.0.0"
 	id("com.github.breadmoirai.github-release") version "2.2.9"
 	id("com.github.ben-manes.versions") version "0.21.0"
+	id("com.install4j.gradle") version "8.0"
+}
+
+install4j {
+	installDir = file("/opt/install4j8/")
```

Comment:
This should be a gradle property

Reply:
What do you mean ?

Reply:
Somet hing in settings.gradle.kts ?

Reply:
no, something in gradle.properties, because it will vary depending on your device - see `properties`

Reply:
Sure.

Reply:
Property with 507a37006627acfbf8849e3c39cf13d24ef6a3a9